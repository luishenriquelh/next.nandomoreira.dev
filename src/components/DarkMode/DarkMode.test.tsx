import React from 'react';
import { render } from '@testing-library/react';
import { DarkMode } from '.';

describe('<DarkMode />', () => {
  test('Render component', () => {
    const componentTree = render(<DarkMode />);
    expect(componentTree).toMatchSnapshot();
  });
});
