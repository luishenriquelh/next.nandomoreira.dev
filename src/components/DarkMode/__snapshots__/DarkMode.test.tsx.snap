// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DarkMode /> Render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none transparent;
  border: none;
  bottom: 40px;
  cursor: pointer;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  display: inline-block;
  font-size: 16px;
  height: 40px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  line-height: 40px;
  padding: 0;
  position: fixed;
  right: 40px;
  width: 40px;
}

<body>
    <div>
      <button
        class="c0"
      >
        <svg
          fill="none"
          height="36"
          width="36"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M31.113 19.567c-1.016.27-2.064.407-3.115.407-3.203 0-6.21-1.245-8.47-3.504a12.038 12.038 0 01-3.095-11.584 1.5 1.5 0 00-1.838-1.835 15.02 15.02 0 00-6.733 3.873c-5.847 5.847-5.847 15.364 0 21.215a14.904 14.904 0 0010.608 4.394 14.893 14.893 0 0010.605-4.393 15.011 15.011 0 003.874-6.736 1.502 1.502 0 00-1.836-1.837zm-4.158 6.452a11.92 11.92 0 01-8.484 3.515 11.93 11.93 0 01-8.487-3.517c-4.677-4.678-4.677-12.292 0-16.97a11.884 11.884 0 013.09-2.225 15.04 15.04 0 004.335 11.772 14.956 14.956 0 0011.772 4.336 12.055 12.055 0 01-2.226 3.089z"
            fill="#272643"
          />
        </svg>
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="DarkModestyles__DarkMode-sc-12towz7-0 kLOegO"
    >
      <svg
        fill="none"
        height="36"
        width="36"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M31.113 19.567c-1.016.27-2.064.407-3.115.407-3.203 0-6.21-1.245-8.47-3.504a12.038 12.038 0 01-3.095-11.584 1.5 1.5 0 00-1.838-1.835 15.02 15.02 0 00-6.733 3.873c-5.847 5.847-5.847 15.364 0 21.215a14.904 14.904 0 0010.608 4.394 14.893 14.893 0 0010.605-4.393 15.011 15.011 0 003.874-6.736 1.502 1.502 0 00-1.836-1.837zm-4.158 6.452a11.92 11.92 0 01-8.484 3.515 11.93 11.93 0 01-8.487-3.517c-4.677-4.678-4.677-12.292 0-16.97a11.884 11.884 0 013.09-2.225 15.04 15.04 0 004.335 11.772 14.956 14.956 0 0011.772 4.336 12.055 12.055 0 01-2.226 3.089z"
          fill="#272643"
        />
      </svg>
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
